### PR TITLE
Update Ch1 with editorial changes

### PR DESCRIPTION
**Pull request by jni**
This doesn't address figure captions and references, which we can't really do from our current build process.

Additionally, I've changed a couple of uses of `arr.sum()` to `np.sum(arr)`.

---
This change is [Reviewable](https://reviewable.io/reviews/elegant-scipy/elegant-scipy/317)

### CODE REVIEW

**Comment by jni** [comment]
This is ready to merge (I'll address changes to each chapter in separate PRs).

File: markdown/ch1.markdown [hunk 1]
```diff
@@ -57,59 +57,62 @@ def rpkm(counts, lengths):
...
 next, differential expression can distinguish different kinds of cancer.
 
 <img src="../figures/differential_gene_expression.png"/>
 <!-- caption text="Gene expression" -->
 
 The state-of-the-art technology to measure mRNA is RNA sequencing (RNAseq).
-RNA is extracted from a tissue sample, for example from a biopsy from a patient, *reverse transcribed* back into DNA (which is more stable), and then read out using chemically modified bases that glow when they are incorporated into the DNA sequence.
-Currently, high-throughput sequencing machines can only read short fragments (approximately 100 bases is common). These short sequences are called “reads”.
+RNA is extracted from a tissue sample (e.g., from a biopsy from a patient), *reverse transcribed* back into DNA (which is more stable), and then read out using chemically modified bases that glow when they are incorporated into the DNA sequence.
+Currently, high-throughput sequencing machines can only read short fragments (approximately 100 bases is common). These short sequences are called “reads.”
```
Comment (stefanv) [inline markdown/ch1.markdown]: Same as I saw before; is this punctuation correct?

Comment (jni) [reply markdown/ch1.markdown]: Sadly, yes. As a computer scientist, I hate it, but it's what English style guides all recommend: punctuation inside the quote marks.

Comment (stefanv) [reply markdown/ch1.markdown]: That's just wrong!!

Comment (i-s-h-i-m) [reply markdown/ch1.markdown]: In terms of grammar and punctuation - that depends on America vs Britain English language. For example:

http://www.thepunctuationguide.com/british-versus-american-style.html

http://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks

https://www.theguardian.com/media/mind-your-language/2011/may/19/mind-your-language-punctuation-quotations

Comment (jni) [reply markdown/ch1.markdown]: Thanks @i-s-h-i-m! That was both enlightening and confusing. =D I had to go pull out books by different authors to verify some of the claims, and they checked out! I had never consciously noticed British authors using single quotes by default, but sure enough, there it was!

My new policy is: do as I please and then let the editor sort it out. =P

Comment (stefanv) [reply markdown/ch1.markdown]: Well, I'd be!

**Pull request merged**